src/System.Net.NetworkInformation/src/System/Net/NetworkInformation/StringParsingHelpers.Connections.cs

```diff
@@ -139,7 +139,7 @@ public static IPEndPoint[] ParseActiveUdpListenersFromFiles(string udp4File, str
         // Parsing logic for local and remote addresses and ports, as well as socket state.
         internal static TcpConnectionInformation ParseTcpConnectionInformationFromLine(string line)
         {
-            StringParser parser = new StringParser(line, ' ', true);
+            StringParser parser = new StringParser(line, ' ', skipEmpty: true);
```

Comment:
You could also just switch the optional parameters in the overload (make the new one last). Up to you.